pages/badges/badge-single.jsx

```diff
@@ -133,8 +133,10 @@ var BadgePage = React.createClass({
     if (data.earned) { status = Badge.achieved; }
     if (data.pending) { status = Badge.pending; }
 
-    var reqs = bdata.require_claim_evidence_description;
-    var matched = reqs.match(/((\d+(\.)?)?[^.]+)/g);
+    var evidenceRegex = /((\d+([.\!?])?)?[^.]+)/g;
```

Comment:
I honestly have no idea! It seems like the "evidence" and "criteria" on Credly are inputted as a single long string containing multiple points. From what I understand from the current code, this regex is meant to take that string and break into multiple strings based on each point. All of this is based on assumptions though that it was decided that there would be a set input format for those two fields and that we want to split them up on the client to display them better. It would be good for us to chat about this though!